src/rules/converters/ban-types.ts

```diff
@@ -1,10 +1,37 @@
 import { RuleConverter } from "../converter";
 
-export const convertBanTypes: RuleConverter = () => {
+export const convertBanTypes: RuleConverter = tslintRule => {
+    type ConvertBanTypeArgument = {
+        message: string;
+    };
+
+    const bannedTypesObj: { [key: string]: ConvertBanTypeArgument | null } = {};
```

Comment:
A few naming nits for this file:

* "`Obj`" is unnecessary in `bannedTypesObj`; could you rename to `bannedTypes`?
* `typ` isn't very clear and I was confused at first as to what it means; could you rename to `bannedType` or similar?